### DISCUSSION TITLE
Support ISO SQL/PGQ, GPM

### DISCUSSION THREAD

**Discussion by annainfo** [duckdb/duckdb#5933] [Ideas]
https://www.gqlstandards.org/ July, 2022 – GQL Status Update
"SQL/PGQ is interesting to people focused on GQL because the Graph Pattern Matching (GPM) language in SQL/PGQ is essentially identical to GPM in GQL. Since the technical work on SQL/PGQ is now stable, GPM in GQL is now stable."

https://www.iso.org/standard/79473.html
ISO/IEC DIS 9075-16
Information technology — Database languages SQL — Part 16: Property Graph Queries (SQL/PGQ)

Proposal to support SQL/PGQ, GPM so that users of DuckDB can use that new query mechanism.

**Comment by Alex-Monahan** [comment]
Howdy! Thanks for the idea! 

Is this something that you or other folks you know would be able to help with?
